package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "monday-ui-react-core",
-  "version": "0.3.50",
+  "version": "0.3.51",
```

Comment:
need to check:
-best way to attribute a11y (support role/aria label?)
-better animation for shimmer than margin?